test/integration/query-interface.test.js

```diff
@@ -454,6 +454,89 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
     });
   });
 
+  describe('removeColumn', function() {
+    describe('(without a schema)', function() {
+      beforeEach(function() {
+        return this.queryInterface.createTable('users', {
+          id: {
+            type: DataTypes.INTEGER,
+            primaryKey: true,
+            autoIncrement: true
+          },
+          firstName: {
+            type: DataTypes.STRING,
+            defaultValue: 'Someone'
+          },
+          lastName: {
+            type: DataTypes.STRING
+          }
+        });
+      });
+
+      it('should be able to remove a column with a default value', function() {
+        return this.queryInterface.removeColumn('users', 'firstName', {
+          logging: log
+        }).then(function() {
+          expect(count).to.be.at.least(1);
+          count = 0;
```

Comment:
Ugh this should all really be refactored to use spys, but i can't fault you for following the rest of the suite :)